include/mercury/kw/kw.h

```diff
@@ -24,16 +26,28 @@ class KanoWorld
     /**
      * \brief Sets the system wallpaper.
      */
+    KanoWorld();
+    ~KanoWorld();
+
     bool login(string username, string password, bool verbose);
     bool refresh_token(string token, bool verbose);
 
     string get_hostname(string config_filename);
     string get_refresh_header(string token);
 
-    static size_t write_function(void *ptr, size_t size, size_t nmemb, void *stream);
+    static size_t write_function(void *ptr, size_t size, size_t nmemb, void *user_data);
 
-    bool am_i_logged_in(void);
+    bool am_i_logged_in(bool verbose);
+    string get_token(void);
```

Comment:
We are using C++, don't need to have `(void)` everywhere, just `string get_token()`, etc.